examples/postgres/vagga.yaml

```diff
@@ -6,31 +6,76 @@ containers:
   ubuntu:
     setup:
     - !Ubuntu xenial
-    - !Install
-      - postgresql-9.5
+    # Use fixed user id and group id for postgres, because in some cases
+    # we may need to upgrade (rebuild) a postgres container, but keep the data
+    # on a `!Persistent` volume still usable. User ids in ubuntu packages are
+    # not guaranteed to be same on every installation.
+    #
+    # The command-lines are from the postgres-common package except
+    # added --uid 200 --gid 200
+    - !Sh |
+        addgroup --system --gid 200 postgres
+        adduser --uid 200 --system --home /data --no-create-home \
+            --shell /bin/bash --group --gecos "PostgreSQL administrator" \
+            postgres
+    - !Install [postgresql-9.5]
     - !EnsureDir /data
     environ:
       PG_PORT: 5433   # Port of host to use
       PG_DB: vagga-test
-      PG_USER: vagga
-      PG_PASSWORD: vagga
       PGDATA: /data
       PG_BIN: /usr/lib/postgresql/9.5/bin
     volumes:
-      /data: !Tmpfs
-        size: 100M
-        mode: 0o700
+      /data: !Persistent
+        name: postgres
+        owner-uid: 200
+        owner-gid: 200
+        init-command: _pg-init
+      /run: !Tmpfs
+        subdirs:
+          postgresql: { mode: 0o777 }  # until we have user, group options
 
 commands:
-  psql: !Command
+
+  _pg-init: !Command
+    description: Init postgres database
+    container: ubuntu
+    user-id: 200
+    group-id: 200
+    run: |
+      set -ex
+      ls -la /data
+      $PG_BIN/pg_ctl initdb
+      $PG_BIN/pg_ctl -w -o '-F --port=$PG_PORT -k /tmp' start
+      $PG_BIN/psql -h 127.0.0.1 -p $PG_PORT \
```

Comment:
Maybe, with [createuser](https://www.postgresql.org/docs/9.5/static/app-createuser.html) this will look prettier?
`$PG_BIN/createuser -h 127.0.0.1 -p $PG_PORT vagga`